pkg/storage/storage.go

```diff
@@ -116,6 +117,10 @@ func (p *Storage) GetContainerMetrics(pods ...apitypes.NamespacedName) ([]api.Ti
 }
 
 func (p *Storage) Store(batch *MetricsBatch) error {
+	// No node has been scraped. Return so the previous updated cache remains intact
```

Comment:
I'm not sure about leaving not clearing cache. HPA and VPA are prepared to handle missing data, but returning stale data could lead to bad decisions.